lte/gateway/c/core/oai/tasks/mme_app/mme_config.cpp

```diff
@@ -1513,12 +1514,17 @@ int mme_config_parse_string(const char* config_string,
               config_pP->nas_config.prefered_integrity_algorithm[i] =
                   EIA2_128_ALG_ID;
             else
-              config_pP->nas_config.prefered_integrity_algorithm[i] =
-                  EIA0_ALG_ID;
+              Fatal("Unrecognized algorithm in ORDERED_SUPPORTED_INTEGRITY_ALGORITHM_LIST\n");
           }
 
+          char default_integrity_algorithm;
+          if (num == 0)
```

Comment:
Isn't this case already handled by the `AssertFatal` above?

Reply:
Thank you.

// This check was from before I added the AssertFatal. 

This code is now:
```
          char default_integrity_algorithm = config_pP->nas_config.prefered_integrity_algorithm[num-1];
           for (i = num; i < 8; i++) {
             config_pP->nas_config.prefered_integrity_algorithm[i] = default_integrity_algorithm;
           }
```